ch-covidcertificate-backend-verifier/ch-covidcertificate-backend-verifier-ws/src/test/java/ch/admin/bag/covidcertificate/backend/verifier/ws/controller/KeyControllerV2Test.java

```diff
@@ -376,6 +380,48 @@ public void keysListAndUpdatesUpToTest() throws Exception {
         assertEquals(String.valueOf((int) verifierDataService.findMaxDscPkId()), upTo);
     }
 
+    @Test
```

Comment:
Could you provide some comments describing what you are testing at which assertion?